tasks/managed/apply-mapping/apply-mapping.yaml

```diff
@@ -22,22 +22,104 @@ spec:
       type: string
       description: Fail the task if the resulting snapshot contains 0 components
       default: "false"
+    - name: ociStorage
+      description: The OCI repository where the Trusted Artifacts are stored.
+      type: string
+      default: "empty"
+    - name: ociArtifactExpiresAfter
+      description: Expiration date for the trusted artifacts created in the
+        OCI repository. An empty string means the artifacts do not expire
+      type: string
+      default: "1d"
+    - name: trustedArtifactsDebug
+      description: Flag to enable debug logging in trusted artifacts. Set to a non-empty string to enable.
+      type: string
+      default: ""
+    - name: orasOptions
+      description: oras options to pass to Trusted Artifacts calls
+      type: string
+      default: ""
+    - name: sourceDataArtifact
+      type: string
+      description: Location of trusted artifacts to be used to populate data directory
+      default: ""
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+    - name: dataDir
```

Comment:
Now that the stepactions combine dataDir and subdirectory, can't we do the same thing in the tasks? I see
```
        - name: workDir
          value: $(params.dataDir)/$(params.subdirectory)
```
which makes me think probably?

Reply:
I have managed to remove as many subdirectory params as I could. A couple of tasks still needed the parameter to help when running unit tests.